### ISSUE TITLE
www.rainwizzard@blogspot.com

### ISSUE THREAD

**Issue by Chodum91** [Chodum91/Papa.Legba.NB#15]
(!(image)(https://github.com/LarryExpo/DarkRose75/assets/93821375/ad18b932-c0ad-4ec6-b9f2-a77ab472e470))()( "ns": "yt", "el": "detailpage", "cpn": "ibTZmf8be-LCc7D9", "ver": 2, "cmt": "255.88", "fmt": "18", "fs": "0", "rt": "36.958", "euri": "", "lact": 2, "cl": "530756126", "mos": 0, "state": "8", "volume": 100, "subscribed": "1", "cbrand": "apple", "cbr": "Facebook Webview", "cbrver": "16.4", "c": "MWEB", "cver": "2.20230511.01.00", "cplayer": "UNIPLAYER", "cmodel": "iphone", "cos": "iPhone", "cosver": "16_4_1", "cplatform": "MOBILE", "hl": "en_GB", "cr": "CA", "len": "1669.561", "fexp":[]() "23983296,24004644,24007246,24080738,24135310,24255165,24405913,24415864,24439361,24451437,24468691,24488209,24499534,24516157,24532854,24556992,24559644,39323074", "feature": "youtu.be", "muted": "0", "docid": "QFg6z02b-VU", "ei": "BQxhZJqPNsL18gS0_rnACQ", "plid": "AAX7qdG2KN-pjqvK", "referrer": "https://accounts.google.com/", "sdetail": "f:youtu.be,p:accounts.google.com", "sourceid": "r", "of": "Sc5K1b8_B1VIixt1Kr3XbQ", "vm": "CAQQARgCOjJBQ00wQ1lqaEM0RlB4UnE2dE00Q2xMT3Z0eUw2RnF4V2VFb2V1c0JMTnNNbllTNEd3Z2JaQVBta0tESllRUE5CN25WLXVPdVJ3ZjNLZ2ZZM3lNUl9hUlJLcVJ4NzlubndkcjJrTXR6M3g1dThHS2F1S04xVHhmY0VHS0hBdTJiYjlhMk9pZDUxOVZWRHlnaAE", "vct": "255.881", "vd": "1669.561", "vpl": "217.549-229.951,249.656-255.881,1176.000-1180.658", "vbu": "0.000-1669.561", "vpa": "0", "vsk": "0", "ven": "0", "vpr": "1", "vrs": "4", "vns": "1", "vec": "null", "vemsg": "", "vvol": "1", "vdom": "1", "vsrc": "1", "vw": "390", "vh": "219", "ismb": 20920000, "relative_loudness": "3.260", "optimal_format": "360p", "user_qual": 0, "release_version": "youtube.player.web_20230509_02_RC00", "debug_videoId": "QFg6z02b-VU", "0sz": "false", "op": "", "yof": "false", "dis": "", "gpu": "Apple_GPU", "debug_playbackQuality": "medium", "debug_date": "Sun May 14 2023 13:28:27 GMT-0300 (Atlantic Daylight Saving Time)" ) - Google Search
( "ns": "yt", "el": "detailpage", "cpn": "ibTZmf8be-LCc7D9", "ver": 2, "cmt": "255.88", "fmt": "18", "fs": "0", "rt": "36.958", "euri": "", "lact": 2, "cl": "530756126", "mos": 0, "state": "8", "volume": 100, "subscribed": "1", "cbrand": "apple", "cbr": "Facebook Webview", "cbrver": "16.4", "c": "MWEB", "cver": "2.20230511.01.00", "cplayer": "UNIPLAYER", "cmodel": "iphone", "cos": "iPhone", "cosver": "16_4_1", "cplatform": "MOBILE", "hl": "en_GB", "cr": "CA", "len": "1669.561", "fexp": "23983296,24004644,24007246,24080738,24135310,24255165,24405913,24415864,24439361,24451437,24468691,24488209,24499534,24516157,24532854,24556992,24559644,39323074", "feature": "youtu.be", "muted": "0", "docid": "QFg6z02b-VU", "ei": "BQxhZJqPNsL18gS0_rnACQ", "plid": "AAX7qdG2KN-pjqvK", "referrer": "https://accounts.google.com/", "sdetail": "f:youtu.be,p:accounts.google.com", "sourceid": "r", "of": "Sc5K1b8_B1VIixt1Kr3XbQ", "vm": "CAQQARgCOjJBQ00wQ1lqaEM0RlB4UnE2dE00Q2xMT3Z0eUw2RnF4V2VFb2V1c0JMTnNNbllTNEd3Z2JaQVBta0tESllRUE5CN25WLXVPdVJ3ZjNLZ2ZZM3lNUl9hUlJLcVJ4NzlubndkcjJrTXR6M3g1dThHS2F1S04xVHhmY0VHS0hBdTJiYjlhMk9pZDUxOVZWRHlnaAE", "vct": "255.881", "vd": "1669.561", "vpl": "217.549-229.951,249.656-255.881,1176.000-1180.658", "vbu": "0.000-1669.561", "vpa": "0", "vsk": "0", "ven": "0", "vpr": "1", "vrs": "4", "vns": "1", "vec": "null", "vemsg": "", "vvol": "1", "vdom": "1", "vsrc": "1", "vw": "390", "vh": "219", "ismb": 20920000, "relative_loudness": "3.260", "optimal_format": "360p", "user_qual": 0, "release_version": "youtube.player.web_20230509_02_RC00", "debug_videoId": "QFg6z02b-VU", "0sz": "false", "op": "", "yof": "false", "dis": "", "gpu": "Apple_GPU", "debug_playbackQuality": "medium", "debug_date": "Sun May 14 2023 13:28:27 GMT-0300 (Atlantic Daylight Saving Time)" )()()
Admriss Investments Ltd

https://youtube.com/@White-Boy-Papalega.N.B

on December 09, 2022

<?xml version="1.0" encoding="utf-8"?> <rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:slash="http://purl.org/rss/1.0/modules/slash/"> <channel> <title>Canadian Juice Monsters</title> <description>Canada's largest bodybuilding &amp; steroid discussion forum</description> <pubDate>Thu, 09 Mar 2023 09:06:38 +0000</pubDate> <lastBuildDate>Thu, 09 Mar 2023 09:06:38 +0000</lastBuildDate> <generator>Canadian Juice Monsters</generator> <link>https://canadianjuicemonsters.net/forums/</link> <atom:link rel="self" type="application/rss+xml" href="https://canadianjuicemonsters.net/forums/forums/-/index.rss"/> <item> <title>Dianabol Oral First Cycle</title> <pubDate>Thu, 09 Mar 2023 08:28:07 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/dianabol-oral-first-cycle.72588/</link> <guid isPermaLink="false">72588</guid> <author>invalid@example.com (PSYCHOS1S)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/anabolic-steroid-discussion.48/"><![CDATA[Anabolic Steroid Discussion]]></category> <dc:creator>PSYCHOS1S</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">I&#039;ve ordered some dbol tablets (25mg/tablet) and they should be coming soon. I have a few questions about the dosage and length of use.<br /> <br /> I&#039;m 5&#039;9 120 lbs. I&#039;m trying to get to at least 150 lbs<br /> <br /> I am trying to put on as much weight as possible, in the next 2 to 3 months. I have setup a diet plan to follow, which includes 3,400 - 3,900 calories per day.<br /> I have a home gym with some generic bulking routines I found online, not sure if I can post the link to the source (against rules?).<br /> I have...<br /> <br /> <a href="https://canadianjuicemonsters.net/forums/threads/dianabol-oral-first-cycle.72588/" class="link link--internal">Read more</a></div>]]></content:encoded> <slash:comments>51</slash:comments> </item> <item> <title>Anybody microdose cialis(tadalifil)</title> <pubDate>Thu, 09 Mar 2023 07:45:56 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/anybody-microdose-cialis-tadalifil.72583/</link> <guid isPermaLink="false">72583</guid> <author>invalid@example.com (robertdeuce)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/anabolic-steroid-discussion.48/"><![CDATA[Anabolic Steroid Discussion]]></category> <dc:creator>robertdeuce</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">Was watching a recent mark bell podcast ep with a doctor from Marek health who is big on microdosing cialis daily(2.5mg) so I figured I&#039;d look into it and was pleasantly surprised of the list of benefits. Anybody here have experience with this?<br /> <br /> -improves lean mass<br /> -improves androgen receptor expression<br /> -positive bp effect<br /> -significantly improve test to estrogen ratio<br /> - prostate enlargement protection<br /> -reduction in c reactive protein(inflammation)markers<br /> -cognitive enhancement via cerebral...<br /> <br /> <a href="https://canadianjuicemonsters.net/forums/threads/anybody-microdose-cialis-tadalifil.72583/" class="link link--internal">Read more</a></div>]]></content:encoded> <slash:comments>9</slash:comments> </item> <item> <title>Orals Sublingually</title> <pubDate>Thu, 09 Mar 2023 07:39:05 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/orals-sublingually.72594/</link> <guid isPermaLink="false">72594</guid> <author>invalid@example.com (1degree)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/anabolic-steroid-discussion.48/"><![CDATA[Anabolic Steroid Discussion]]></category> <dc:creator>1degree</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">I&#039;ve read some research that taking orals sublingually has a better effect, I can see if they are tablets, but what if they are capsules (powder) ?</div>]]></content:encoded> <slash:comments>0</slash:comments> </item> <item> <title>My intro</title> <pubDate>Thu, 09 Mar 2023 07:30:33 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/my-intro.72593/</link> <guid isPermaLink="false">72593</guid> <author>invalid@example.com (Hollywood604)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/introduce-yourself.55/"><![CDATA[Introduce Yourself]]></category> <dc:creator>Hollywood604</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">Hello all, long-time lurker here - <br /> <br /> 42 years old, 190lbs 5&#039;11&quot;<br /> <br /> I&#039;ve been training since my early teens with a couple of years off when life got a hold of me with too many women and too many pills <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="smilie smilie--sprite smilie--sprite6" alt=":cool:" title="Cool :cool:" loading="lazy" data-shortname=":cool:" /><br /> <br /> My first introduction to running gear was at the ripe age of 17. Sustanon 250 and Anadrol with no PCT. Yup, not proud of that one in so many ways.<br /> <br /> Back then we used to run veterinarian test called Testorona 200 that we smuggled back from Mexico. We&#039;d also use horse gear called Finaplex...<br /> <br /> <a href="https://canadianjuicemonsters.net/forums/threads/my-intro.72593/" class="link link--internal">Read more</a></div>]]></content:encoded> <slash:comments>0</slash:comments> </item> <item> <title>Faer labs vs forza vs Olympia gold</title> <pubDate>Thu, 09 Mar 2023 06:04:04 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/faer-labs-vs-forza-vs-olympia-gold.72592/</link> <guid isPermaLink="false">72592</guid> <author>invalid@example.com (Dowhitelines)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/anabolic-steroid-discussion.48/"><![CDATA[Anabolic Steroid Discussion]]></category> <dc:creator>Dowhitelines</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">Which is the better one out of those 3 only got access too</div>]]></content:encoded> <slash:comments>0</slash:comments> </item> <item> <title>My Introduction</title> <pubDate>Thu, 09 Mar 2023 03:45:17 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/my-introduction.72590/</link> <guid isPermaLink="false">72590</guid> <author>invalid@example.com (Hamilton_Wannabe)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/anabolic-steroid-discussion.48/"><![CDATA[Anabolic Steroid Discussion]]></category> <dc:creator>Hamilton_Wannabe</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">Hello all!<br /> <br /> <br /> <br /> Long time lurker, first time poster as I am getting way more interested in anabolics then I thought I ever would!<br /> <br /> <br /> <br /> About me: 30 years old, 6’1, steady 210lbs. Have not taken body fat % in a while but I’m definitely not all muscle and have some weight shredding to do.<br /> <br /> <br /> <br /> I would describe myself as very active. Always grew up in sports &amp; lifting, currently work in construction and work out 5-7 days / week.<br /> <br /> <br /> <br /> I did my first cycle in high school messing around without much of...<br /> <br /> <a href="https://canadianjuicemonsters.net/forums/threads/my-introduction.72590/" class="link link--internal">Read more</a></div>]]></content:encoded> <slash:comments>12</slash:comments> </item> <item> <title>Irregular heart beat and steroids</title> <pubDate>Thu, 09 Mar 2023 03:30:54 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/irregular-heart-beat-and-steroids.72586/</link> <guid isPermaLink="false">72586</guid> <author>invalid@example.com (Cmor)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/introduce-yourself.55/"><![CDATA[Introduce Yourself]]></category> <dc:creator>Cmor</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">I’m considering starting my first cycle, I’ve been training naturally for 7 years. One of my main concerns however is I have an irregular heartbeat. I’m curious on what precautions I should be taking to limit risk (blood thinner, bp meds etc.) and make sure that I do this as safely as possible.<br /> Thank you for any input and information</div>]]></content:encoded> <slash:comments>14</slash:comments> </item> <item> <title>Carbohydrate cycling</title> <pubDate>Thu, 09 Mar 2023 02:08:20 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/carbohydrate-cycling.72584/</link> <guid isPermaLink="false">72584</guid> <author>invalid@example.com (Lexy)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/diet-nutrition-forum.52/"><![CDATA[Diet & Nutrition Forum]]></category> <dc:creator>Lexy</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">I would like to start a body recomposition program using carb cycling but I am unclear as how to calculate the amount of carbs for high and low days and how many high/low days I should have during the week. I&#039;ve lost 10 pounds of weight the last 5 wks on a 500 cal deficit diet ( eating 2500 cal/d ). I kept my macros at 250g of protein, 250g of carbs and 55g of fat. I train push, pull, legs 6 days a week and push a weighted sled in the morning for mobility work ( weights in the pm). Started...<br /> <br /> <a href="https://canadianjuicemonsters.net/forums/threads/carbohydrate-cycling.72584/" class="link link--internal">Read more</a></div>]]></content:encoded> <slash:comments>19</slash:comments> </item> <item> <title>second cycle advice</title> <pubDate>Thu, 09 Mar 2023 02:08:06 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/second-cycle-advice.72567/</link> <guid isPermaLink="false">72567</guid> <author>invalid@example.com (AntonChigurh)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/anabolic-steroid-discussion.48/"><![CDATA[Anabolic Steroid Discussion]]></category> <dc:creator>AntonChigurh</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">Hi, I’m 28 years old &amp; did a cycle a couple of years ago of Test E &amp; <a href="http://www.steroid.com/Deca.php" target="_blank" class="link link--external" rel="nofollow ugc noopener">Deca</a> &amp; <a href="http://www.steroid.com/Dianabol.php" target="_blank" class="link link--external" rel="nofollow ugc noopener">Dbol</a> , I had amazing gains with it, most of it I lost during inactivity over the years.<br /> <br /> However I’ve been back training so far for a while again now &amp; wanting to do another cycle of Test E 500mg/week, most likely again with just Dbol this time as I’m just wanting to bulk. <br /> <br /> any advice would be greatly appreciated on how long I should keep my cycle and PCT process, thanks!</div>]]></content:encoded> <slash:comments>35</slash:comments> </item> <item> <title>Does CJM have an app?</title> <pubDate>Thu, 09 Mar 2023 00:14:17 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/does-cjm-have-an-app.63985/</link> <guid isPermaLink="false">63985</guid> <author>invalid@example.com (Mr Destruction)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/suggestions-feedback.50/"><![CDATA[Suggestions / Feedback]]></category> <dc:creator>Mr Destruction</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">Hey guys, is there an mobile app for CJM?</div>]]></content:encoded> <slash:comments>22</slash:comments> </item> <item> <title>Newport</title> <pubDate>Wed, 08 Mar 2023 23:26:14 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/newport.72585/</link> <guid isPermaLink="false">72585</guid> <author>invalid@example.com (ShadowBanGhost)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/underground-steroid-lab-reviews.73/"><![CDATA[Underground Steroid Lab Reviews]]></category> <dc:creator>ShadowBanGhost</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">Not looking for the source just a brand g2g. Has anyone tried Newport pharmacy?</div>]]></content:encoded> <slash:comments>8</slash:comments> </item> <item> <title>Global pharma</title> <pubDate>Wed, 08 Mar 2023 23:23:47 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/global-pharma.72591/</link> <guid isPermaLink="false">72591</guid> <author>invalid@example.com (deeknowsgymstuff)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/underground-lab-pictures.74/"><![CDATA[Underground Lab Pictures]]></category> <dc:creator>deeknowsgymstuff</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">My mild summer cycle has arrived. I gotta say the global line is much nicer looking in person than pics. I always say i wouldnt hesitate to use them, so i will give them a run. I have used the orgenon sust amps in the past from the same place and they were great. Many say those are not legit but are the same quality or same product as the global test ampules, if so it will be good. I generally dont like ampules but since i plan on doing 1cc shots twice a week its fine. If i had to crack 3...<br /> <br /> <a href="https://canadianjuicemonsters.net/forums/threads/global-pharma.72591/" class="link link--internal">Read more</a></div>]]></content:encoded> <slash:comments>2</slash:comments> </item> <item> <title>first cycle - test only</title> <pubDate>Wed, 08 Mar 2023 22:14:58 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/first-cycle-test-only.72589/</link> <guid isPermaLink="false">72589</guid> <author>invalid@example.com (Delusional)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/anabolic-steroid-discussion.48/"><![CDATA[Anabolic Steroid Discussion]]></category> <dc:creator>Delusional</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">never done a cycle, and was looking into a tbol cycle, but the feedback i received helped me realize how stupid that is.<br /> Although im only 18, im still slightly interested in a test only cycle. Im a university track athlete, and track season is approaching. so i&#039;m looking for that athletic edge, and also the increase in size.<br /> Would a test only cycle with proper PCT be harmful to my health going forward? Even if it was a low dosage (200-300mg range)</div>]]></content:encoded> <slash:comments>14</slash:comments> </item> <item> <title>Anavar responsiveness?</title> <pubDate>Wed, 08 Mar 2023 21:56:14 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/anavar-responsiveness.72572/</link> <guid isPermaLink="false">72572</guid> <author>invalid@example.com (Yoni Kim)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/anabolic-steroid-discussion.48/"><![CDATA[Anabolic Steroid Discussion]]></category> <dc:creator>Yoni Kim</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">From my knowledge, your cant be unresponsive to steroids but is it possible? I&#039;ve been on Anavar for the past 4 weeks, a 40mg cycle, 4 pills, 1 every 4 hours. I got it from a reputable source and every time I&#039;ve asked someone about it they said it was legit but I&#039;m seeing 0 results on weight or strength, not even better pumps I legit feel nothing. I also haven&#039;t been feeling any side effects but that might just be because I am large and the steroid itself is weak. diet has been normal 4...<br /> <br /> <a href="https://canadianjuicemonsters.net/forums/threads/anavar-responsiveness.72572/" class="link link--internal">Read more</a></div>]]></content:encoded> <slash:comments>35</slash:comments> </item> <item> <title>Travelling with Testosterone</title> <pubDate>Wed, 08 Mar 2023 21:47:41 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/travelling-with-testosterone.72557/</link> <guid isPermaLink="false">72557</guid> <author>invalid@example.com (swolegorilla)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/suggestions-feedback.50/"><![CDATA[Suggestions / Feedback]]></category> <dc:creator>swolegorilla</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">Travelling end of month from Toronto to Costa Rica for 2 weeks. Wondering if it’s safe to pack a vial and syringes in the checked baggage. (Self administered)<br /> <br /> Any info will be helpful</div>]]></content:encoded> <slash:comments>33</slash:comments> </item> <item> <title>New member</title> <pubDate>Wed, 08 Mar 2023 20:59:54 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/new-member.72563/</link> <guid isPermaLink="false">72563</guid> <author>invalid@example.com (Delusional)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/introduce-yourself.55/"><![CDATA[Introduce Yourself]]></category> <dc:creator>Delusional</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">Hello everyone, i am 18, 5&#039;10, 173, and about 14% bf<br /> ive been working out for 3 years, but only seriously this last 1 year ish<br /> played sports my whole life, and am taking track very seriously right now<br /> i have never used steroids before, but am looking into running my first cycle of turinabol</div>]]></content:encoded> <slash:comments>20</slash:comments> </item> <item> <title>Eager to learn and succeed</title> <pubDate>Wed, 08 Mar 2023 20:01:15 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/eager-to-learn-and-succeed.72587/</link> <guid isPermaLink="false">72587</guid> <author>invalid@example.com (PSYCHOS1S)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/introduce-yourself.55/"><![CDATA[Introduce Yourself]]></category> <dc:creator>PSYCHOS1S</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">Hello<br /> <br /> I&#039;m a hard gainer who is new to bodybuilding and I look forward to reading through the forum and learning. There&#039;s so much to learn and I&#039;m so far behind but I&#039;m dedicated and now is the time to really commit myself and work hard.<br /> <br /> I&#039;m trying to put on some weight, and the situation is dire. I&#039;m 5&#039;9 120 lbs lean.<br /> <br /> New plan though; I&#039;ve quit smoking ciggs it&#039;s been 7 months so that has increased my appetite a bit, not much but it&#039;s a start; I&#039;m never hungry it&#039;s a real issue with me...<br /> <br /> <a href="https://canadianjuicemonsters.net/forums/threads/eager-to-learn-and-succeed.72587/" class="link link--internal">Read more</a></div>]]></content:encoded> <slash:comments>10</slash:comments> </item> <item> <title>Brown nipple discharge</title> <pubDate>Wed, 08 Mar 2023 19:14:48 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/brown-nipple-discharge.72582/</link> <guid isPermaLink="false">72582</guid> <author>invalid@example.com (BanditBodydroppa)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/anabolic-steroid-discussion.48/"><![CDATA[Anabolic Steroid Discussion]]></category> <dc:creator>BanditBodydroppa</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">Yes... as the title states... brown coloured nipple discharge...<br /> <br /> So my nipples were a little puffy today and I&#039;ve heard people mention they get discharge occasionally. So I said what the hell, I&#039;m gonna squeeze em and see what happens, lol. Low and behold this dark brown liquid starts coming out. It wasn&#039;t thick, it was more on the watery side.<br /> <br /> Has anyone experienced this and what the hell is causing it?!<br /> <br /> I know this is definitely an off topic and will probably head in the wrong direction...<br /> <br /> <a href="https://canadianjuicemonsters.net/forums/threads/brown-nipple-discharge.72582/" class="link link--internal">Read more</a></div>]]></content:encoded> <slash:comments>45</slash:comments> </item> <item> <title>an old one but still funny..</title> <pubDate>Wed, 08 Mar 2023 17:28:15 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/an-old-one-but-still-funny.70871/</link> <guid isPermaLink="false">70871</guid> <author>invalid@example.com (Renegade-Rabbit)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/general-discussion.49/"><![CDATA[General Discussion]]></category> <dc:creator>Renegade-Rabbit</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">morning all hope the day is going well so far</div>]]></content:encoded> <slash:comments>61</slash:comments> </item> <item> <title>More interesting discussion</title> <pubDate>Wed, 08 Mar 2023 15:23:48 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/more-interesting-discussion.65382/</link> <guid isPermaLink="false">65382</guid> <author>invalid@example.com (66Chevy)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/general-discussion.49/"><![CDATA[General Discussion]]></category> <dc:creator>66Chevy</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">Some interesting theories and reading. I have some thoughts, but would like to hear some different takes on this...as CJM has some smart fellers.<br /> <br /> <br /> <br /> 	 	<div class="bbCodeBlock bbCodeBlock--unfurl js-unfurl fauxBlockLink" 		data-unfurl="true" data-result-id="958" data-url="https://www.armstrongeconomics.com/world-news/conspiracy/are-the-planning-id2020-as-mandatory-implants-for-all-as-the-solution-to-the-crisis/" data-host="www.armstrongeconomics.com" data-pending="false"> 		<div class="contentRow"> 			 				<div class="contentRow-figure contentRow-figure--fixedSmall js-unfurl-figure"> 					 						<img src="https://www.armstrongeconomics.com/wp-content/uploads/2020/03/Digital-Microchip.jpg" alt="www.armstrongeconomics.com" data-onerror="hide-parent"/> 					 				</div> 			 			<div class="contentRow-main"> 				<h3 class="contentRow-header js-unfurl-title"> 					<a href="https://www.armstrongeconomics.com/world-news/conspiracy/are-the-planning-id2020-as-mandatory-implants-for-all-as-the-solution-to-the-crisis/" 						class="link link--external fauxBlockLink-blockLink" 						target="_blank" 						rel="nofollow ugc noopener" 						data-proxy-href=""> 						Are They Planning ID2020 as Mandatory Implants for All as the Solution to the Crisis? | Armstrong Economics 					</a> 				</h3> 				<div class="contentRow-snippet js-unfurl-desc">There is a very concerning proposal running around in Washington. The technology already exists for pets which is an electronic microchip or transponder. A subcutaneous electronic microchip allows you to identify your pet. It is no larger than a grain of rice and only takes a few minutes for...</div> 				<div class="contentRow-minor contentRow-minor--hideLinks"> 					<span class="js-unfurl-favicon"> 						 							<img src="https://www.armstrongeconomics.com/wp-content/themes/customtheme/images/favicon.png" alt="www.armstrongeconomics.com" class="bbCodeBlockUnfurl-icon" 								data-onerror="hide-parent"/> 						 					</span> 					www.armstrongeconomics.com 				</div> 			</div> 		</div> 	</div></div>]]></content:encoded> <slash:comments>4755</slash:comments> </item> <item> <title>First time using steroids</title> <pubDate>Wed, 08 Mar 2023 14:11:48 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/first-time-using-steroids.72562/</link> <guid isPermaLink="false">72562</guid> <author>invalid@example.com (Delusional)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/anabolic-steroid-discussion.48/"><![CDATA[Anabolic Steroid Discussion]]></category> <dc:creator>Delusional</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">I dont know where to buy steroids from. I hvae never used steroids before, and am looking to run a cycle of turinabol. i live in Saskatchewan and have been working out for 3 years, almost 4.<br /> Ive heard mixed reviews about newport pharma recently, but they also take e-transfer for payment and lots of posts on this wesbite say e-transfer is sketchy. any help?<br /> <br /> **edit<br /> my bad i didnt read the rules, also posted an introduction, apologies</div>]]></content:encoded> <slash:comments>8</slash:comments> </item> <item> <title>The CJM science thread</title> <pubDate>Wed, 08 Mar 2023 03:33:26 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/the-cjm-science-thread.72553/</link> <guid isPermaLink="false">72553</guid> <author>invalid@example.com (Rolls roydz)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/general-discussion.49/"><![CDATA[General Discussion]]></category> <dc:creator>Rolls roydz</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">We have threads about movies, politics, women 🤣 and a lot of stuff.<br /> <br /> I will propose this thread for sharing science videos, channel or whatever interesting nerd cool video.<br /> <br /> I will start with:<br /> <br /> 	<div> 		<a href="https://m.youtube.com/@veritasium" 			class="link link--external" 			target="_blank" 			rel="nofollow ugc noopener" 			data-proxy-href=""> 			https://m.youtube.com/@veritasium 		</a> 	</div> 	<div> 		<a href="https://m.youtube.com/@kurzgesagt" 			class="link link--external" 			target="_blank" 			rel="nofollow ugc noopener" 			data-proxy-href=""> 			https://m.youtube.com/@kurzgesagt 		</a> 	</div></div>]]></content:encoded> <slash:comments>10</slash:comments> </item> <item> <title>Mystery bottles from Syn</title> <pubDate>Wed, 08 Mar 2023 02:32:00 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/mystery-bottles-from-syn.72390/</link> <guid isPermaLink="false">72390</guid> <author>invalid@example.com (Kanon)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/underground-lab-pictures.74/"><![CDATA[Underground Lab Pictures]]></category> <dc:creator>Kanon</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">Got these mystery bottles from Syn in my last order. Possible addition to their lineup? <br /> <br /> Any guesses what it is ?</div>]]></content:encoded> <slash:comments>45</slash:comments> </item> <item> <title>Hi there!</title> <pubDate>Wed, 08 Mar 2023 00:38:22 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/hi-there.72581/</link> <guid isPermaLink="false">72581</guid> <author>invalid@example.com (LostSignal)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/introduce-yourself.55/"><![CDATA[Introduce Yourself]]></category> <dc:creator>LostSignal</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">Hi everyone,<br /> I have been a longtime lurker, and have learned so much from doing so. <br /> I appreciate this community and all the knowledge it provides to many who wish to take their fitness to a new level. <br /> I have recently began prescribed TRT and have seen amazing gains, even on a relatively low dose. <br /> I am middle aged now, and I am gaining muscle faster than when in my 20’s. <br /> When I ask my children if daddy is getting bigger muscles, they always say “no, you look the same daddy”. Kids, lol...<br /> <br /> <a href="https://canadianjuicemonsters.net/forums/threads/hi-there.72581/" class="link link--internal">Read more</a></div>]]></content:encoded> <slash:comments>9</slash:comments> </item> <item> <title>Skin care routines</title> <pubDate>Tue, 07 Mar 2023 23:23:06 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/skin-care-routines.67067/</link> <guid isPermaLink="false">67067</guid> <author>invalid@example.com (LiRoyle)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/general-discussion.49/"><![CDATA[General Discussion]]></category> <dc:creator>LiRoyle</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">Whats going on guys, wanted to start a discussion regarding skin care on/off cycle and what everyones experiences are regarding skin care. What do you guys use? <br /> <br /> Currently i just used basic cleansers and moisturizer daily and keep cleansing wipes in my truck for when i forget/get real dirty at work. Seems to work decently but still get some breakouts every once in a while. Experimenting currently with saliatic acid creams for the blackheads on my nose, wich never seem to go away or change...<br /> <br /> <a href="https://canadianjuicemonsters.net/forums/threads/skin-care-routines.67067/" class="link link--internal">Read more</a></div>]]></content:encoded> <slash:comments>31</slash:comments> </item> <item> <title>Strength/Powerlifting Log</title> <pubDate>Tue, 07 Mar 2023 22:49:00 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/strength-powerlifting-log.58780/</link> <guid isPermaLink="false">58780</guid> <author>invalid@example.com (rhynoplanet)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/logs-and-journals.116/"><![CDATA[Logs and Journals]]></category> <dc:creator>rhynoplanet</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">I have decided to post my strength log. I train for powerlifting in the Masters division since I am 42 and currently weigh 230 lbs so compete in the 242 class. Will probably eventually cut down to compete in the 220s<br /> <br /> Today I did<br /> <br /> Low Bar squats barx8, 135x8, 225x5, 285x5, 335x4, 385x3, 385x3, 440x2, 440x2, 440x3 Did an extra rep the last set of 440, was only supposed to do 2. Felt easy today.<br /> Deadlift 275x3, 330x4, 385x3, 385x3, 440x3, 440x3, 440x3, 440x3. Felt easy and light<br /> Paused squats...<br /> <br /> <a href="https://canadianjuicemonsters.net/forums/threads/strength-powerlifting-log.58780/" class="link link--internal">Read more</a></div>]]></content:encoded> <slash:comments>1705</slash:comments> </item> <item> <title>Cheatmeal thread</title> <pubDate>Tue, 07 Mar 2023 21:51:37 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/cheatmeal-thread.72528/</link> <guid isPermaLink="false">72528</guid> <author>invalid@example.com (deeknowsgymstuff)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/diet-nutrition-forum.52/"><![CDATA[Diet & Nutrition Forum]]></category> <dc:creator>deeknowsgymstuff</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">A safe place where you can post your cheatmeals judgement free! 😉<br /> <br /> Todays cheatmeal - perogies, chorico, beef patty, bear paws, kit kats. 😁<br /> <a href="https://canadianjuicemonsters.net/forums/attachments/20230226_110929-jpg.25791/" 		target="_blank"><img src="https://canadianjuicemonsters.net/forums/data/attachments/22/22645-81fdaf9b452d3b986e62340d8ae4b227.jpg" 		class="bbImage " 		style="" 		alt="20230226_110929.jpg" 		title="20230226_110929.jpg" 		width="150" height="204" loading="lazy" /></a></div>]]></content:encoded> <slash:comments>37</slash:comments> </item> <item> <title>Doctor question timeline, your tough ?</title> <pubDate>Tue, 07 Mar 2023 21:22:58 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/doctor-question-timeline-your-tough.72297/</link> <guid isPermaLink="false">72297</guid> <author>invalid@example.com (Lincoln.n)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/post-cycle-therapy-pct-discussions.70/"><![CDATA[Post Cycle Therapy (PCT) Discussions]]></category> <dc:creator>Lincoln.n</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">Long story short, I am coming off a long cruise of testosterone 175mg/week.<br /> <br /> When talking to my doctor, he question right away why would I wait 4 weeks before starting Clomid/nolva. <br /> <br /> And is answer made me question a lot my way of thinking (keep in mind he is not an endo). He basically said that supra physiological dose of test will suppress you for a longer period of time because blood level needs to go down. (same thing with highly suppressive 19-nor). But in theory : Physiological dose of...<br /> <br /> <a href="https://canadianjuicemonsters.net/forums/threads/doctor-question-timeline-your-tough.72297/" class="link link--internal">Read more</a></div>]]></content:encoded> <slash:comments>16</slash:comments> </item> <item> <title>Guidance</title> <pubDate>Tue, 07 Mar 2023 20:03:41 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/guidance.72579/</link> <guid isPermaLink="false">72579</guid> <author>invalid@example.com (1degree)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/introduce-yourself.55/"><![CDATA[Introduce Yourself]]></category> <dc:creator>1degree</dc:creator> <content:encoded><![CDATA[<div class="bbWrapper">Hello, everyone! I&#039;m new to the forum, but I&#039;ve been reading posts here for at least five years. Finally, I&#039;ve decided to become an active member. I&#039;ve run two cycles in the past and am starting a new one soon, but I never got any blood work from a doctor. I want to proceed with my upcoming cycle properly, so I&#039;m wondering: what should I be looking for when getting blood work? Should I see my regular family doctor (I live in Canada), or do I need to find a different type of doctor? My cycle...<br /> <br /> <a href="https://canadianjuicemonsters.net/forums/threads/guidance.72579/" class="link link--internal">Read more</a></div>]]></content:encoded> <slash:comments>12</slash:comments> </item> <item> <title>The hobby thread.</title> <pubDate>Tue, 07 Mar 2023 19:48:59 +0000</pubDate> <link>https://canadianjuicemonsters.net/forums/threads/the-hobby-thread.72068/</link> <guid isPermaLink="false">72068</guid> <author>invalid@example.com (Slartibartfast)</author> <category domain="https://canadianjuicemonsters.net/forums/forums/gentlemen%E2%80%99s-corner.139/"><![CDATA[Gentlemen’s Corner]]></category> <dc:creator>Slartibartfast</dc:creator> <content:encoded><![CDATA[<div class=

￼

"bbWrapper">Just as a change of subject for a bit, I&#039;m pretty sure that some of you have creative outlets that don&#039;t necessarily involve a lot of grunting and sweating, although those are the fun ones, and I&#039;ve seen that <a href="https://canadianjuicemonsters.net/forums/members/1025530/" class="username" data-xf-init="member-tooltip" data-user-id="1025530" data-username="@Legacy">@Legacy</a> is working on his truck and others have mentioned their skills in

building and fixing things. <br /> <br /> So, I thought I&#039;d see if anyone is interested in sharing some of their hobbies or things they&#039;ve created outside of the gym. (Illegitimate children don&#039;t count)<br /> <br /> I do a lot of...<br /> <br /> <a href="https://canadianjuicemonsters.net/forums/threads/the-hobby-thread.72068/" class="link link--internal">Read more</a></div>]]></content:encoded> <slash:comments>50</slash:comments> </item> </channel> </rss>

WWW.RAINWIZZARD@BLOGSPOT.COM
